src/components/Loading/__tests__/loading.test.jsx

```diff
@@ -1,8 +1,12 @@
 import React from 'react';
 import { render } from '@testing-library/react';
 import Loading from '../Loading';
+import { displayIcon } from '../loadingUtil';
 
 describe('Loading', () => {
+    const renderWovenIcon = 'primary';
+    const renderTRIIcon = 'secondary';
+    const logo = (props) => <svg {...props} />;
     it('renders with no label by default', () => {
```

Comment:
nit: Could use a line break between the variable declarations and first test for easier readability.